Felienne/hedy
# Adds types to level 5 and 6 in semantics description

**Description**

adds types to level 5 and 6 in semantics description. the types are the same as for previous levels, but "if" and "if else" are added.I  gave both type boolean, since the line they will or will not go to is a new command already

**How to test**

See SEMANTICS.md, that's the only file that's changed. See if you think the types for "if" and "if else" being boolean is okay, and if it is correct that all the other types don't change in these levels.
